mod.go

```diff
@@ -108,22 +112,42 @@ func getVCSInfoForModule(m *listModule) (*moduleVCSInfo, error) {
 	return info, nil
 }
 
-// moduleDir returns the path to the directory to be
-// used for storing the module with the given path.
-func moduleDir(module string) string {
-	// TODO decide what color this bikeshed should be.
+// moduleDir returns the path to the directory to be used for storing the
+// module with the given path, as well as the filepath to be used in a replace
+// directive. If $GOHACK is set then it will be used. A relative $GOHACK will
+// be interpretted relative to main module directory.
+func moduleDir(module string) (path string, replPath string, err error) {
+	modfp := filepath.FromSlash(module)
+	// Performing a FromSlash here allows /-delimited path values in environment
+	// variables on Windows
```

Comment:
Turns out this conversation is moot in any case 😄 

Because the value we get from the environment variable will ultimately be passed with `filepath.Join`, which calls `filepath.Clean` on the result. 

So I think this is fine. 

Alternatively we can call `filepath.Clean` to perhaps better express the intent?

Reply:
In which case, let's just remove the explicit FromSlash call because it's redundant?

Reply:
👍